Client/src/ArmoniK.DevelopmentKit.Client.Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -99,6 +100,12 @@ public class BaseClientSubmitter<T>
   [CanBeNull]
   protected ILogger<T> Logger { get; set; }
 
+
+  /// <summary>
+  ///   The number of chunk to split the payloadsWithDependencies
+  /// </summary>
+  private int ChunkSubmitSize { get; set; }
```

Comment:
If it is private, it does not really make sense to have those accessors. Juste remove them and use the proposed name